doc/user/content/sql/create-sink.md

```diff
@@ -50,7 +50,8 @@ Field | Value type | Description
 ### AS OF
 
 `AS OF` is the specific point in time to start emitting all events for a given `SINK`. If you don't
-use `AS OF`, Materialize will pick a timestamp itself.
+use `AS OF`, Materialize will pick a timestamp itself. `timestamp_expression` supports the same
```

Comment:
Rather than adding this...

> `timestamp_expression` supports...

Can you add the `timestamp_expression` details from https://materialize.io/docs/sql/tail/#syntax to this page's syntax section? This makes it sound like this supports some kind of special expression, which it just needs something that's in time relative to the Unix epoch.

Reply:
makes sense, thanks!  updated